### PR TITLE
Radioactive decay test node

### PR DESCRIPTION
**Pull request by TheDoctorRAB**
This is a test node for radioactive decay.

### CODE REVIEW

File: radioactive-decay/content.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# <div style ="text-align: right">Radioactive Decay</div>
```
Comment (gonuke) [inline radioactive-decay/content.md]: Probably shouldn't have this HTML markup here - not sure where it came from?

File: radioactive-decay/content.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+#### Learning objective
+Basic understanding of radioactive decay
+
+<hr>
```
Comment (gonuke) [inline radioactive-decay/content.md]: More HTML... pure markdown will have none of this, maybe your editor?

Comment (gonuke) [reply radioactive-decay/content.md]: P.S. I guess this is why we'll need a style guide :)

File: radioactive-decay/exercises/q_radioactive_decay.yml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+uid: q-radioactive decay 
+title: Radioactive Decay
+type: Derivation
+options:
+  text: Derive the Poisson distribution from the binomial distribution.
```
Comment (gonuke) [inline radioactive-decay/exercises/q_radioactive_decay.yml]: (meta-comment: This may be a place where reviews can get kind of contentious.... )  Does this assessment actually test the learning objective?  This assessment is a purely mathematical/statistical operation.

File: radioactive-decay/info.yml [hunk 4]
```diff
@@ -0,0 +1,15 @@
+---
+uid: radioactive_decay 
+title: Radioactive decay 
+prerequisites:
+  - basic statistics and probability
+  - undergraduate level  
+learning_objectives:
+  - Basic understanding of radioactivity
```
Comment (gonuke) [inline radioactive-decay/info.yml]: This is a broad learning objective, perhaps more specificity?  (also a contrib/style guide issue)

File: radioactive-decay/exercises/special-relativity-example.yml [hunk 5]
```diff
@@ -0,0 +1,18 @@
+---
```
Comment (gonuke) [inline radioactive-decay/exercises/special-relativity-example.yml]: This question/file should be removed.

File: .gitignore [hunk 6]
```diff
@@ -1,3 +1,4 @@
 _site/
 .sass-cache/
 .jekyll-metadata
+*~
```
Comment (gonuke) [inline .gitignore:4]: Ideally this change would be in a separate PR - It's clearly valuable, but probably doesn't need to be tied up with the issues that may arise here.

**Review by gonuke — COMMENTED** [review]
I consider these early reviews an important part of the process of learning how this will work.  Since we don't yet have complete style guides or contribution guides, things will be subject to a lot of subjectivity.  Hopefully we can use these early reviews to boil down to some consensus.

File: radioactive-decay/content.md (hunk 1 above)
Comment (TheDoctorRAB) [reply radioactive-decay/content.md]: I think you can get rid of this pull request. I initially did it as practice to get used to git, etc., and I'm not done with this node anyway. But I'll remove html.

File: radioactive-decay/exercises/q_radioactive_decay.yml (hunk 3 above)
Comment (TheDoctorRAB) [reply radioactive-decay/exercises/q_radioactive_decay.yml]: This was also for practice. The complete problem has them compute the 1st and 2nd moments of the distribution and figure out what it means. (They're equal!). You can see the problem in the SIIP repository.

File: radioactive-decay/content.md (hunk 1 above)
Comment (gonuke) [reply radioactive-decay/content.md]: I think it's great practice for both making PRs but also for practicing reviewing.  That's one aspect of this that will be a little different for all of us, I think, despite our experience with code reviews

File: .gitignore (hunk 6 above)
Comment (TheDoctorRAB) [reply .gitignore:4]: I didn't realize that everything got pushed at once, and the pull request was just all the files.

**Comment by katyhuff** [comment]
This comment is not a review, but is perhaps related to your question about images: On Saturday of this week, I'm going to try hard to get the website to render these nodes. The way you have linked the images looks right, but lots of things about linking and metadata may have to be tweaked once I start setting these up to build properly. Will let you know when I know more on that.

**Comment by katyhuff** [comment]
@TheDoctorRAB Ideally, the style guide should be in a different pull request. You can do this by creating a new branch (e.g. "style-guide"), which will keep such commits out of this PR.

**Comment by TheDoctorRAB** [comment]
I was trying to and I screwed it up. I thought it would be a new pull request because this commit only contained the style guide. I'm still trying to get up to speed.

**Comment by katyhuff** [comment]
No big deal. Just best practice not a showstopper. Chapter 21 of the
effective computation in physics book book, should be collaboration, which
has an example related to having an experimental branch of code.

**Comment by TheDoctorRAB** [comment]
I think I fixed it right. I'm going to create a new pull request later then from my new radioactive-decay branch so everything is cleaned up.

**Comment by gonuke** [comment]
This has been replaced/superseded by #35
